docs/data/charts/axis/GroupedAxesStyling.js

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { axisClasses } from '@mui/x-charts/ChartsAxis';
+
+export default function GroupedAxesStyling() {
+  return (
+    <BarChart
+      sx={{
+        [`& [data-group-index="1"] .${axisClasses.tick}`]: {
+          stroke: 'rgb(255, 180, 34)',
+        },
+        [`& [data-group-index="0"] .${axisClasses.tickLabel}`]: {
+          fill: 'gray',
```

Comment:
This isn't very visible in dark mode:

<img width="853" height="339" alt="image" src="https://github.com/user-attachments/assets/293a4d39-32cf-4928-86f2-c2c064ae459b" />

Reply:
I don't think they are meant to be very visible. I could use other color though